src/app.js

```diff
@@ -520,15 +520,7 @@ var run = function () {
     } else {
       url = 'https://ethereum.github.io/solc-bin/bin/' + version;
     }
-    var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
@yann300 true means using worker, so with removing it in every case it is using in the browser thread.